sklearn/svm/_classes.py

```diff
@@ -992,7 +996,8 @@ class SVR(RegressorMixin, BaseLibSVM):
 
     _impl = 'epsilon_svr'
 
-    def __init__(self, kernel='rbf', degree=3, gamma='scale',
+    @_deprecate_positional_args
+    def __init__(self, *, kernel='rbf', degree=3, gamma='scale',
```

Comment:
I'd consider `kernel` being positional in SVMs... but SVC has `C` in the way.

Reply:
Maybe that's something we can do once the deprecation cycle is over, because there would be no risk of breaking user code?